helpers.js

```diff
@@ -0,0 +1,51 @@
+const fs = require('fs')
+
+const config = require('./config')
+
+const loggerImpl = (level, message) => console.log(`🦒 [${level}] ${message}`)
+
+const helpers = {}
+
+helpers.logger = {
+    debug: (message) => loggerImpl('debug', message),
+    info: (message) => loggerImpl('info', message),
+    warn: (message) => loggerImpl('warn', message),
+    error: (message) => loggerImpl('❌error', message)
```

Comment:
Very cool, but in a time stressing exercise, don't bother, just use `console.log`

Reply:
sure, I'll remove it and use console.log instead